uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/Api/DocumentTest.php

```diff
@@ -9,6 +9,11 @@
 class Api_DocumentTest extends CiviUnitTestCase {
```

Comment:
I know this not part of this ticket/PR, but the correct location of this file should be tests/phpunit/api/v3/DocumentTest.php and the class name should be api_v3_DocumentTest. 

If you could fix that it would be great!